packages/webapp/src/components/Task/AbandonTask/index.jsx

```diff
@@ -113,7 +115,21 @@ const PureAbandonTask = ({
 
       {hasAssignee && (
         <>
-          <Main style={{ marginBottom: '24px' }}>{t('TASK.ABANDON_TASK_DURATION')}</Main>
```

Comment:
We can make this:

`<Main style={{ marginBottom: '24px' }} tooltipContent={t('TASK.ABANDON_TASK_DURATION')}>{t('TASK.ABANDON_TASK_DURATION')}</Main>`

Reply:
That is much simpler and cleaner. Thanks!